src/axolotl/processing_strategies.py

```
@@ -454,6 +454,63 @@ def process_labels(self, input_ids):
+    """
+
+    def __init__(
+        self,
+        processor: ProcessorMixin,
+        chat_template: Optional[str] = None,
+        image_size: int | tuple[int, int] | None = None,
+        image_resize_algorithm: Resampling | None = None,
+    ):
+        super().__init__(processor, chat_template, image_size, image_resize_algorithm)
+
+        self.image_token = "<|image|>"
```

Ruff (0.14.10)

Possible hardcoded password assigned to: "image_token"

(S105)